lib/Doctrine/deploy/AddNGIs.php

```diff
@@ -51,6 +56,10 @@
             $securityEmail = (string) $value;
         }
 
+        if((string) $key == "OBJECT_ID") {
```

Comment:
This is the line causing the Codacy fail - however it is in the same format as the other if statements above. Should I fix?

Reply:
As it's a new line of code, could you address the issue. A seperate PR that fixes the issue for the file as a whole would also be most welcome :)

Reply:
done in #428